src/javascripts/ng-admin/Crud/column/maReferenceColumn.js

```diff
@@ -0,0 +1,28 @@
+function maReferenceColumn() {
+    return {
+        restrict: 'E',
+        scope: {
+            field: '&',
+            value: '&',
+            datastore: '&'
+        },
+        link: {
+            pre: function(scope) {
+                const value = scope.value();
+                scope.field = scope.field();
+                scope.targetEntity = scope.field.targetEntity();
+                scope.targetField = scope.field.targetField();
+                const identifierName = scope.targetEntity.identifier().name()
+                scope.referencedEntry = scope.datastore()
+                    .getEntries(scope.targetEntity.uniqueId + '_values')
+                    .filter(entry => entry.values[identifierName] == value)
```

Comment:
Assuming that the APIs we fetch are consistent (i.e. that the type of `comment.post_id` is the same as the type of `post.id`) is pretty brave, but I think we'd rather be tolerant to ease compatibility. So e.g. when `comment.post_id` is `'1'` and `post.id` is `1`, the current code works.